runner/src/server/plugins/engine/pageControllers/MultiPageController.ts

```diff
@@ -0,0 +1,35 @@
+import { FormData, FormSubmissionErrors } from "../types";
+import { PageController } from "./PageController";
+import config from "server/config";
+import nunjucks from "nunjucks";
+
+export class MultiPageController extends PageController {
+  get viewName() {
+    return "multi-page";
+  }
+  getViewModel(formData: FormData, errors?: FormSubmissionErrors) {
+    const viewModel = super.getViewModel(formData, errors);
+    const { showContinueButton, startPageNavigation } = this.pageDef;
+    if (config.allowUserTemplates) {
+      if (startPageNavigation.previous) {
+        startPageNavigation.previous.labelText = nunjucks.renderString(
+          startPageNavigation.previous.labelText,
+          { ...formData }
+        );
+      }
+      if (startPageNavigation.next) {
+        startPageNavigation.next.labelText = nunjucks.renderString(
+          startPageNavigation.next.labelText,
+          { ...formData }
+        );
+      }
```

Comment:
I didn't realise that there was a govukPagination component. can we set the schema to match govukPagination? that way we just pass everything straight through to the component.

I suppose it would mean more things are "hardcoded" but that's ok?

Reply:
So that is how it works pretty much, what this part is doing though is rendering templating variables in the label text. Although now I'm thinking about it maybe we shouldn't allow templating in these bits for the time being?